src/main/fc/settings.yaml

```diff
@@ -72,7 +72,7 @@ tables:
     values: ["BATTERY", "CELL"]
     enum: osd_stats_min_voltage_unit_e
   - name: osd_video_system
-    values: ["AUTO", "PAL", "NTSC", "HD"]
+    values: ["AUTO", "PAL", "NTSC", "HDZERO", "DJIWTF"]
```

Comment:
I thought about this and I think this would be the easiest for the end user. Everybody knows what kind of goggles they have, but I would guess that almost nobody knows what size their OSD is.
Also, HDZERO must have that option selected for their OSD to work (must send the MSP_SET_OPTIONS subcommand followed by a 0 and a 1).
DJI (with wtf) will support any of those resolutions (and it needs a 2 instead of a 1 for the biggest screen)
Right now both of those goggles use the MSP_SET_OPTIONS subcommand, any new goggles might use something else to turn on the OSD, so they would need their own handling.
I'm open to changing them to something else, but we also want to keep it easy for the end user to know what to select.

Reply:
I agree with Jeff here. I think that it will be easier for end users just to select the system they're using. Most people will not know that they are using an X character by X character grid for their OSD. How many characters does Avatar use? Is that information even anywhere? I am guessing it's about 54 x 20, but who knows?

Reply:
@MrD-RC Your correct... Avatar is 54 x 20.  I counted the characters horizontally and vertically on the screen, just to check.

Reply:
Do we know how Avatar is going to send commands to the VTX?
I was just looking through the HDZero code and they have 3 resolutions defined, so I bumped up the number for DJI wtfos by 1 and I'm talking with the MSP OSD guys to do the same.